### DISCUSSION TITLE
Symfony 5 entity validation

### DISCUSSION THREAD

**Discussion by kostya1000** [symfony/symfony#43152] [Q&A]
Hello everybody.
I had some experience with Symfony 4. Now I'm doing a new project on Symfony 5.  I've found new behavior in entity validation in forms.
Symfony 4 automatically adds validator to entity field based on Doctrine column parameters. For example, if an entity column is not null, there will be a NotBlank constraint.
But now it doesn't work in Symfony 5. I have an entity with a few not null fields, but no validators appear in the form.
This thing doesn't work anymore or am I doing something wrong?

**Comment by jfsenechal** [comment]
Hello

It's explain here

https://symfony.com/doc/current/doctrine.html#validating-objects

**Reply by kostya1000** [reply]
I've read this. It is written there:

> Symfony introspects the Doctrine mapping configuration to infer some validation rules. For example, given that the name property can’t be null in the database, a NotNull constraint is added automatically to the property (if it doesn’t contain that constraint already).

But it doesn't append validation rules based on the Doctrine mapping configuration. PropertyInfo component is installed.
Have you tried?
